src/Collection.php

```diff
@@ -78,6 +78,20 @@ class Collection
     private $typeMap;
     private $writeConcern;
 
+    /**
+     * Returns whether we are currently in a transaction
+     *
+     * @param array  $options  Command options
+     * @return bool
+     */
+    private function isInTransaction(array $options)
```

Comment:
PedantryTest may complain about this, since it checks that private methods appear after all public and protected methods. Consider moving below `public function withOptions()`.